src/logic/map_objects/tribes/tribe_descr.cc

```diff
@@ -213,49 +213,42 @@ TribeDescr::TribeDescr(const Widelands::TribeBasicInfo& info,
 		log_info("┃    Ships: ");
```

Comment:
There is nothing following the `:` any more for these. I'd be fine with getting rid of the details and just having `Loading barbarians ... took %ums`.

Rest looks good.